composer.json

```diff
@@ -30,7 +30,6 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "composer-runtime-api": "^2.0",
-        "composer/package-versions-deprecated": "^1.8",
```

Comment:
this seems not be used and our required packages seems migrated to the official composer api.